modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/PlacementDriverManagerTest.java

```diff
@@ -321,6 +449,45 @@ private void checkLeaseCreated(TablePartitionId grpPartId) throws InterruptedExc
         }, 10_000));
     }
 
+    /**
+     * Checks that the lease for the group is accepted.
+     *
+     * @param grpPartId Replication group id.
+     * @throws InterruptedException If the waiting is interrupted.
+     */
+    private void checkAccepted(TablePartitionId grpPartId) throws InterruptedException {
+        assertTrue(waitForCondition(() -> {
+            var leaseFut = metaStorageManager.get(fromString(PLACEMENTDRIVER_PREFIX + grpPartId));
+
+            var leaseEntry = leaseFut.join();
+
+            if (leaseEntry != null && !leaseEntry.empty()) {
+                Lease lease = ByteUtils.fromBytes(leaseEntry.value());
+
+                return lease.isAccepted();
+            }
+
+            return false;
+        }, 10_000));
+    }
+
+    /**
+     * Reads a leas from Meta storage. When you call this method the lease is expected presenting in Meta storage.
```

Comment:
typos

- leas -> lease
- the lease is expected presenting in Meta storage. -> the lease is expected to be present in Meta storage.